src/leappto/actor_cli.py

```diff
@@ -0,0 +1,154 @@
+import os
+import subprocess
+from argparse import ArgumentParser
+from pprint import pprint
+from subprocess import PIPE
+
+import argcomplete
+import snactor.executors
+
+from snactor import loader
+from snactor import registry
+
+ACTOR_DIRECTORY = '/usr/share/leapp/actors'
+VERSION = "0.2-dev"
+
+
+def _add_identity_options(cli_cmd, context=''):
+    if context:
+        cli_cmd.add_argument('--' + context + '-identity', default=None,
+                             help='Path to private SSH key for the ' + context + ' machine')
+        cli_cmd.add_argument('--' + context + '-user', default="root", help='Connect as this user to the ' + context)
+    else:
+        cli_cmd.add_argument('--identity', default=None, help='Path to private SSH key')
+        cli_cmd.add_argument('--user', '-u', default="root", help='Connect as this user')
+
+
+def _port_spec(arg):
```

Comment:
why not define this under +def make_argument_parser(): ?

Reply:
I don't know why it should be under it - There's nothing that requires it to be under it - Just because you can create them locally it doesn't have to be - And there's no reason to have it within the argument parser